1_For_Loops.ipynb

```diff
@@ -202,7 +202,7 @@
   },
```

Comment:
<p>I want you to think of every variable you have created here and try to explain to yourself what is its purpose and why do you need it. after that think if you can improve your function :)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week04-beckyrosenfeld/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='108'/>